From afbe6c6ee713519263305d7e2d91cc9f1379afe2 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 3 Apr 2023 20:41:40 +0000
Subject: [PATCH] IWYU: add stdint.h for various int types in //base

Bug: 957519
Change-Id: I750de80f0d27d93ef3383052ddb4eefd5ee13abe
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4394430
Reviewed-by: Daniel Cheng <dcheng@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#1125558}
---
 base/debug/profiler.h       | 1 +
 base/strings/string_piece.h | 1 +
 2 files changed, 2 insertions(+)

diff --git a/base/debug/profiler.h b/base/debug/profiler.h
index 5f0deadd5d3..035affc7c4d 100644
--- a/base/debug/profiler.h
+++ b/base/debug/profiler.h
@@ -6,6 +6,7 @@
 #define BASE_DEBUG_PROFILER_H_
 
 #include <stddef.h>
+#include <stdint.h>
 
 #include <string>
 
diff --git a/base/strings/string_piece.h b/base/strings/string_piece.h
index 899b654cbe8..9dde6ec29bb 100644
--- a/base/strings/string_piece.h
+++ b/base/strings/string_piece.h
@@ -22,6 +22,7 @@
 #define BASE_STRINGS_STRING_PIECE_H_
 
 #include <stddef.h>
+#include <stdint.h>
 
 #include <algorithm>
 #include <iosfwd>
